lib/AbstractPromise.php

```diff
@@ -111,4 +111,20 @@ abstract class AbstractPromise implements PromiseInterface
     {
         return $this->state == self::STATE_FULFILLED;
     }
+
+    /**
+     * @param mixed $error
+     * @return string
+     */
+    protected static function stringifyError($error): string
```

Comment:
don't need this, provide original `\Throwable` error

Reply:
you are right, best to let the error bubble up for the implementor to handle. Fixed now